packages/tools/src/tools-routes.tsx

```diff
@@ -23,7 +24,7 @@ export function Router(props: { account: Account; network: Network }) {
 export default function ToolsRoutes() {
   return (
     <UiPage>
-      <PortfolioUiNetworkAccountGuard render={(props) => <Router {...props} />} />
+      <Router />
```

Comment:
This doesn't need to be a separate `Router` component anymore, it can all live in `ToolsRoutes` now,.